include/vcpkg/base/messages.h

```diff
@@ -74,6 +74,12 @@ namespace vcpkg
             return *this;
         }
 
+        LocalizedString& append_nl()
+        {
+            m_data.append("\n");
+            return *this;
+        }
```

Comment:
There used to be a function that did exactly the same but @BillyONeal removed it in #547.

Reply:
I did remove that, but I think @JavierMatosD owns this space now and if he would like an `appendnl` I'm not strongly opposed; the main thing I wanted to be clear in #547 is that 'our newline character is `\n`, do not contort things to get all newlines to be printed to `\n`'.